packages/valory/skills/decision_maker_abci/tests/behaviours/test_base.py

```diff
@@ -299,3 +335,73 @@ def test_get_bet_amount(
             assert e.value == expected_result
         else:
             raise AssertionError("Expected `StopIteration` exception!")
+
+    @pytest.mark.parametrize(
+        "trading_strategy, win_probability, confidence, selected_type_tokens_in_pool, other_tokens_in_pool, bet_fee, weighted_accuracy, token_balance, wallet_balance, expected_result",
+        # fmt: off
+        (
+            # bet amount per threshold strategy
+            ("bet_amount_per_threshold", 0.0, 0.1, 0.1, 0, 0, 0.0, 0, 0, 0),
+            ("bet_amount_per_threshold", 0.0, 0.6, 0.1, 0, 0, 0.0, 0, 0, 60000000000000000),
+            ("bet_amount_per_threshold", 0.0, 0.8, 0.1, 0, 0, 0.0, 0, 0, 100000000000000000),
+            ("bet_amount_per_threshold", 0.0, 0.9, 0.1, 0, 0, 0.0, 0, 0, 1000000000000000000),
+            # kelly criterion no confidence strategy
+            ("kelly_criterion_no_conf", 0.85, 0.0, 100, 200, 1_000_000_000_000_000, 0.75, 50_000_000_000_000_000_000, 0, 243),
```

Comment:
Could make the numbers a bit more readable, e.g. ` 50_000_000_000_000_000_000== int(5e19)`